benchmarks/JavaScript/richards.js

```diff
@@ -22,7 +22,7 @@ var NO_TASK = null,
 
   DATA_SIZE = 4,
 
-  TRACING = false;
+  TRACING = true;
```

Comment:
Please don't change this. It's not a bug.

Reply:
will revert